java/core/src/java/org/apache/orc/OrcConf.java

```diff
@@ -118,6 +118,11 @@ public enum OrcConf {
       "If the number of distinct keys in a dictionary is greater than this\n" +
           "fraction of the total number of non-null rows, turn off \n" +
           "dictionary encoding.  Use 1 to always use dictionary encoding."),
+  DICTIONARY_MAX_SIZE_IN_BYTES("orc.dictionary.maxSizeInBytes",
+      "hive.exec.orc.dictionary.maxSizeInBytes",
```

Comment:
Let's not introduce this new `Apache Hive` configuration, `hive.exec.orc.dictionary.maxSizeInBytes`. You can simply repeat `orc.dictionary.maxSizeInBytes` here like the other configuration example.

https://github.com/apache/orc/blob/b9035b426070b1a90a6a1b2f02063d34c0eb7353/java/core/src/java/org/apache/orc/OrcConf.java#L98